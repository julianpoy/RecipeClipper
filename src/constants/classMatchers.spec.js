import ClassMatchers from './classMatchers'

describe('classMatchers', () => {
  it('matches snapshot', () => {
    expect(ClassMatchers).toMatchSnapshot()
  })
})

