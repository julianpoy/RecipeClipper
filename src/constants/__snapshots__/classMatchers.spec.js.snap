// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`classMatchers matches snapshot 1`] = `
Object {
  "activeTime": Array [
    Array [
      "activeTime",
      "active-time",
      "prep-time",
      "time-active",
      "time-prep",
    ],
    Array [
      "activeTime",
      "active-time",
      "prep-time",
      "time-active",
      "time-prep",
    ],
  ],
  "description": Array [
    Array [
      "wprm-recipe-summary",
    ],
    Array [],
  ],
  "imageURL": Array [
    Array [
      "wprm-recipe-image",
      "tasty-recipes-image",
      "hero-photo",
      "o-RecipeLead__m-RecipeMedia",
      "recipe-lede-image",
      "recipe-body",
      "recipe__hero",
      "content",
    ],
    Array [
      "recipe-image",
      "hero",
      "recipe-content",
      "recipe-body",
      "recipe-intro",
      "recipe-",
    ],
  ],
  "ingredients": Array [
    Array [
      "wprm-recipe-ingredients-container",
      "wprm-recipe-ingredients",
      "tasty-recipes-ingredients",
      "o-Ingredients",
      "recipe-ingredients",
      "recipe-ingredients-section",
      "ingredientlist",
      "ingredient-list",
    ],
    Array [
      "ingredients",
      "ingredientlist",
      "ingredient-list",
    ],
  ],
  "instructions": Array [
    Array [
      "wprm-recipe-instructions",
      "tasty-recipes-instructions",
      "recipe-directions__list",
      "o-Method",
      "steps-area",
      "recipe-method-section",
      "recipe__list--steps",
      "recipesteps",
      "recipe-steps",
      "instructionlist",
      "instruction-list",
      "preparationsteps",
      "preparation-steps",
      "prepsteps",
      "prep-steps",
      "recipeinstructions",
      "recipe-instructions",
      "recipemethod",
      "recipe-method",
      "directions",
      "instructions",
    ],
    Array [
      "instructionlist",
      "instruction-list",
      "preparationsteps",
      "preparation-steps",
      "recipesteps",
      "recipe-steps",
      "recipemethod",
      "recipe-method",
      "directions",
      "instructions",
    ],
  ],
  "notes": Array [
    Array [
      "notes",
      "recipenotes",
      "recipe-notes",
      "recipefootnotes",
      "recipe-footnotes",
      "recipe__tips",
      "wprm-recipe-notes-container",
    ],
    Array [
      "recipenotes",
      "recipe-notes",
      "recipefootnotes",
      "recipe-footnotes",
    ],
  ],
  "title": Array [
    Array [
      "wprm-recipe-name",
    ],
    Array [
      "recipename",
      "recipe-name",
      "recipetitle",
      "recipe-title",
    ],
  ],
  "totalTime": Array [
    Array [
      "totalTime",
      "total-time",
      "time-total",
    ],
    Array [
      "totalTime",
      "total-time",
      "time-total",
    ],
  ],
  "yield": Array [
    Array [
      "recipe-yield",
      "recipe-servings",
      "yield",
      "servings",
    ],
    Array [
      "yield",
      "servings",
    ],
  ],
}
`;
